app/main.py

```diff
@@ -1 +1,28 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+
+    command = command.split()
+
+    if len(command) < 3 or command[0] != "mv":
+        raise ValueError
+
+    command, source_path, destination_path = command
+
+    separator = os.path.sep
```

Comment:
When the `separator` variable is used only once in the logic, it would be preferable to call `os.path.sep` directly where needed.